apps/miranum-cli/src/app/shared/fs.spec.ts

```diff
@@ -1,7 +1,6 @@
 import { getFile, getFiles } from "./fs";
 import { FileDetails } from "@miranum-ide/miranum-core";
-
-const pathToProject = "resources/my-process-automation-project/";
+import {pathToProject} from "../../../tests/testHelpers";
```

Comment:
```suggestion

const pathToProject = "resources/my-process-automation-project/";
```

In dem Fall würde ich die Testdaten direkt in den Test schreiben.

Reply:
ich finds persönlich schöner wenn es an einer stelle is, dann muss man es nur da ändern, sollte das Projekt verschoben weren
Gibt es irgend nen benefit wenn der path nicht importiert wird?